notebooks/official/workbench/spark/spark_ml.ipynb

```diff
@@ -0,0 +1,1623 @@
+{
```

Comment:
<p>extremely petty nit (sorry) - in American english we say "counter-clockwise" instead of the anti-clockwise that canadians and brits use - normally I wouldn't give a hoot because I know exactly what you mean but I know in our docs we explicitly have American spellings as a standard which is why I'm mentioning it</p><p> </p><p>The rest of this paragraph is so informative though and is like the perfect example of why notebooks are great</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/838/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='54'/>

Reply:
Didn't noticed that I used anti-clockwise! Thanks to let me know!